### PR TITLE
Update the plugin to use new formats and replace payloaders with parsers

### PR DESCRIPTION
**Pull request by Noarkhh**
- Replace MP4.Payload with H264 stream format
- Replace MP4.Payloaders with H264.Parsers

### CODE REVIEW

File: examples/sink.exs [hunk 1]
```diff
@@ -28,26 +29,25 @@ defmodule Example do
...
       |> child(:audio_parser, %Membrane.AAC.Parser{
-        in_encapsulation: :ADTS,
         out_encapsulation: :none
       })
       |> child(:audio_realtimer, Membrane.Realtimer)
       |> via_in(Pad.ref(:audio, 0))
       |> get_child(:rtmp_sink)
     ]
 
-    {[spec: structure, playback: :playing], %{streams_to_end: 2}}
```
Comment (Qizot) [inline examples/sink.exs:50]: Has something changed in the core that `playback: :playing` is no longer required?

**Review by Qizot — APPROVED** [review]
Please post links to PRs that made it possible to get rid of `MP4` payloaders.

File: examples/sink.exs (hunk 1 above)
Comment (varsill) [reply examples/sink.exs:50]: Indeed, it has been removed in https://github.com/membraneframework/membrane_core/releases/tag/v0.12.1

**Review by varsill — APPROVED** [review]

File: examples/sink.exs (hunk 1 above)
Comment (Qizot) [reply examples/sink.exs:50]: Good to know :)

**Pull request merged**